BDPointXamarinBinder.csproj

```diff
@@ -17,7 +17,7 @@
     <MonoAndroidAssetsPrefix>Assets</MonoAndroidAssetsPrefix>
     <AndroidClassParser>class-parse</AndroidClassParser>
     <PackageId>Bluedot.PointSDK.Android</PackageId>
-    <PackageVersion>15.5.1</PackageVersion>
+    <PackageVersion>15.5.2-beta-1</PackageVersion>
```

Comment:
Currently pointing to 15.5.2-beta-1. I will update to 15.5.2 in another PR once it passes QA

Reply:
It should stay as 15.5.2-beta-1 even after QA, until we release native Android SDK 15.5.2 and link to the official 15.5.2.aar, which can be a while.